node/index.ts

```diff
@@ -75,3 +135,229 @@ export class PrivateKey {
     );
   }
 }
+
+export class PreKeyBundle {
+  private readonly nativeHandle: SignalClient.PreKeyBundle;
+
+  constructor(
```

Comment:
I feel weird having some classes use constructors and others use `static new()`. Maybe they should all use `new()`, with the constructor limited to the "from native handle" use case?